packages/html/lib/index.js

```diff
@@ -2,7 +2,7 @@ const { extname } = require('path');
 
 const getFiles = (bundle) => {
   const files = Object.values(bundle).filter(
-    (file) => file.isEntry || file.type === 'asset' || file.isAsset
+    (file) => file.isEntry || (typeof file.type === 'string' ? file.type === 'asset' : file.isAsset)
```

Comment:
Using and accessing `isAsset` is deprecated for assets emitted in Rollup. As it turns out, rollup-plugin-postcss is unfortunately using a non-standard and officially deprecated way of emitting assets by "adding something to the bundle" which only contains `isAsset`, hence the workaround.